wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/person/Person.kt

```diff
@@ -92,6 +92,9 @@ data class Person(
     }
   }
 
+  /**
+   * Represents the type of the phone number: mobile, home or work.
+   */
   enum class PhoneType(private val value: Int) : WireEnum {
     MOBILE(0),
```

Comment:
How about we have a comment on `MOBILE` or any enum value? Maybe not worth it?

Reply:
There's a comment on the `WORK` constant, line 106.